openedx/features/philu_courseware/tests/test_competency_assessment_record_manager.py

```diff
@@ -1,3 +1,6 @@
+"""
+All test cases for competency assessment record manager
+"""
 from django.test import TestCase
 from opaque_keys.edx.keys import UsageKey
```

Comment:
Kindly add a newline between Django imports and project imports.

Reply:
We run isort and do not manage imports ordering and spacing manually. This is the result of isort.